app/qml/MerginSettingDialog.qml

```diff
@@ -0,0 +1,153 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.0
+import QtQuick.Layouts 1.3
+import QtQuick.Dialogs 1.2
+import QtGraphicalEffects 1.0
+import QgsQuick 0.1 as QgsQuick
+
+Dialog {
```

Comment:
Hmm with the desktop build I can't really set the new URL - pressing Enter does not trigger anything and buttons are missing too. This is what I see:

![image](https://user-images.githubusercontent.com/193367/52947573-68917a00-3377-11e9-90c0-7979ac3b3d68.png)

Reply:
Hopefully fixed by proper sizing of a dialog and better key handling - however textField has to have focus to listen on accept. All in new commit.